docs/docs/opinions.md

```diff
@@ -2,11 +2,11 @@
 
 The default project structure reflects certain opinions about how to do collaborative data science work. These opinions grew out of our own experiences with what works and what doesn't. Some of these opinions are about workflows, and others are about tools that can make the process easier. These opinions are discussed below. If you have any thoughts, please [contribute or share them](contributing.md).
 
-## Data analysis is a directed acyclic graph
+### Data analysis is a directed acyclic graph
 
-> Don't _ever_ edit your raw data, especially not manually, and _especially_ not in Excel.
+_Don't ever edit your raw data. Especially not manually. And especially not in Excel._
```

Comment:
Was the un-quote blocking here intentional? I think it's probably fine to keep this as-is, but it breaks the format we use everywhere else of quote-blocking the subtitle.

Reply:
Yeah, it was. This page in particular got feedback that the styles were too busy at first glance.